src/cli.rs

```diff
@@ -25,6 +25,15 @@ pub enum SubCommand {
         /// Write valid config as website-stalker.yaml
         #[arg(long)]
         rewrite_yaml: bool,
+
+        /// Configuration file to use, default website-stalker.yaml
```

Comment:
The default will already be shown by clap, no need to put it in the docs too
```suggestion
        /// Configuration file to use
```